queryx.go

```diff
@@ -216,6 +216,13 @@ func (q *Queryx) Bind(v ...interface{}) *Queryx {
 	return q
 }
 
+// Scan executes the query, copies the columns of the first selected
+// row into the values pointed at by dest and discards the rest. If no rows
+// were selected, ErrNotFound is returned.
+func (q *Queryx) Scan(v ...interface{}) error {
+	return q.Query.Scan(udtWrapSlice(q.Mapper, q.unsafe, v)...)
+}
+
```

Comment:
Wow, do we introduce a new public API in this commit? If so, I'd much prefer to have it sent as a distinct commit, well described what it is and what for.

Reply:
It might seemse like we do, but we don't, since `Queryx` has inherits all public methods from `Query` it was there before, but due to lack of wrapping was not behaving in the same was as other public methods of `Queryx`.

Reply:
I'm not sure if I got it. You mean that it was previously possible to call `Queryx.Scan`, and it was just `Query.Scan` - without calling `udtWrapSlice` on passed arguments?

Reply:
correct

Reply:
Then what you add here is not introducing new public function, but rather fixing an existing one. OK!